nixos/modules/services/misc/hayabusa.nix

```diff
@@ -0,0 +1,50 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
We want to avoid using top-level `with lib;` expression, because it has many known drawback (including increasing eval times!).

- see: #208242